tests/acceptance/stepDefinitions/publicLinkContext.js

```diff
@@ -85,27 +68,32 @@ When(
 )
 
 When(
-  'the public (tries to )open/opens the public link page of the last public link created by user {string} with password {string}',
-  async function(linkCreator, password) {
-    const lastShare = await sharingHelper.fetchLastPublicLinkShare(linkCreator)
-    await client.page.publicLinkFilesPage().navigateAndWaitForPasswordPage(lastShare.token)
-    return client.page.publicLinkPasswordPage().submitPublicLinkPassword(password)
+  'the public uses the webUI to access the last public link created by user {string} with password {string}',
+  function(linkCreator, password) {
+    return loadPublicLinkWithPassword(linkCreator, password, false)
   }
 )
 
 When(
-  'the public uses the webUI to access the last public link created by user {string} with password {string}',
-  async function(linkCreator, password) {
-    const lastShare = await sharingHelper.fetchLastPublicLinkShare(linkCreator)
-    if (lastShare.permissions === sharingHelper.PERMISSION_TYPES.create) {
-      await client.page.filesDropPage().navigateAndWaitForPasswordPage(lastShare.token)
-    } else {
-      await client.page.publicLinkFilesPage().navigateAndWaitForPasswordPage(lastShare.token)
-    }
-    return client.page.publicLinkPasswordPage().submitPublicLinkPassword(password)
+  'the public uses the webUI to access the last public link created by user {string} with password {string} on a new session',
```

Comment:
usually we place "using the webUI" at the end of the sentence, but that sound also fine for me @phil-davis comments?